graph_algorithms/algorithms/install.sh

```diff
@@ -256,10 +256,10 @@ END
 
         # 6. Ask for query mode.
        echo; echo "Please choose query mode:"
-       select mode in "Single Node Mode" "Batch Mode"; do
+       select mode in "Single Node Mode" "Distributed Mode"; do
```

Comment:
Also, elsewhere in install.sh (not in the recent change): "similarity" is misspelled twice.
"similary" --> "similarity"